### PR TITLE
Ports: Add fribidi

### PR DESCRIPTION
**Pull request by n0toose**
This is a part of a greater effort to port SuperTux.

### CODE REVIEW

**Comment by n0toose** [comment]
Hang on, seems like I used the lint script incorrectly.

**Comment by n0toose** [comment]
Should be fixed now, haven't done this in a bit ^^

File: Ports/fribidi/package.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='fribidi'
+version='1.0.13'
+useconfigure='true'
+use_fresh_config_sub='true'
+archive_hash='7fa16c80c81bd622f7b198d31356da139cc318a63fc7761217af4130903f54a2'
+use_fresh_config_sub='true'
```
Comment (ADKaster) [inline Ports/fribidi/package.sh:7]: you've got `use_fresh_config_sub` twice :o

File: Ports/fribidi/package.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+workdir="fribidi-${version}"
+configopts=(
+    "--prefix=${SERENITY_INSTALL_ROOT}/usr/local"
```
Comment (ADKaster) [inline Ports/fribidi/package.sh:10]: --prefix isn't passed to any other meson ports. is it strictly required?

Comment (n0toose) [reply Ports/fribidi/package.sh:10]: The local version worked, but apparently the PR I shipped is slightly problematic in various ways. I will take a look against what other meson ports did and try to clean things up to improve the port's quality.

**Comment by n0toose** [comment]
(Forgot the ReadMe.md, will do another force-push)

Other than that, I fixed the issues that were mentioned in the review, used `" "` only when necessary, fixed unused configuration options and cleaned things up.

**Comment by n0toose** [comment]
Additional note:

The only other recipe using meson the same way that I do is [`wayland`](https://github.com/SerenityOS/serenity/blob/b37f3c86fd5bd4f8f0ee2be69da331620ad5c211/Ports/wayland/package.sh#L5), I am not sure if I left something out.

File: Ports/fribidi/patches/ReadMe.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+```
+-if cc.get_id() == 'gcc' and cc.has_argument('-ansi')
+-  add_project_arguments('-ansi', language: 'c')
+-endif
+```
+
+The inline keyword (C99) is too new for ANSI/C89.
+
+Without this patch, the compiler freaks out because fribidi
+relies on inline functions.
```
Comment (timschumi) [inline Ports/fribidi/patches/ReadMe.md:16]: Please put these three lines into the commit description (of the commit that this patch is based on) and regenerate the patch and README.

Comment (timschumi) [inline Ports/fribidi/patches/ReadMe.md:16]: If this is the case, shouldn't this patch be upstreamable? No need to do that now, just asking.

**Review by timschumi — CHANGES_REQUESTED** [review]

File: Ports/fribidi/patches/ReadMe.md (hunk 3 above)
Comment (n0toose) [reply Ports/fribidi/patches/ReadMe.md:16]: I would upstream it, but I unfortunately do not understand why it exists there in the first place.

**Comment by n0toose** [comment]
I plan to work on this next month. I will close the PR for now so as to be able to reopen it again by myself.

**Pull request closed without merging**